src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine {

```

Comment:
The `doWork()` and `stopWork()` methods are missing in the 'Bulldozer' class. You should override these methods to provide custom behavior for the Bulldozer. 🏗️